var/spack/repos/builtin/packages/openfst/package.py

```diff
@@ -45,10 +45,5 @@ class Openfst(AutotoolsPackage):
     version('1.4.1',  'ca8f1730b9b9b281e515611fa9ae23c0')
     version('1.4.0',  '662367ec91084ffab48ee9b5716de39c')
 
-    @run_before('autoreconf')
-    def check_compilers(self):
-        # Fails to built openfst with %gcc@6: or %intel@16
-        if self.spec.satisfies('%intel@16:') or self.spec.satisfies('gcc@6:'):
-            raise InstallError(
-                'openfst does not support Intel compiler >= 16 and gcc >= 6 '
-            )
+    conflicts('intel@16:')
```

Comment:
Yes. I just fixed this typo. The same issue can be found in #3667 . Since #3667 has been merged, I send a new PR #3673 to fix it.